tests/Message/PurchaseRequestTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Omnipay\Buckaroo\Message;
+
+use Omnipay\Tests\TestCase;
+
+class PurchaseRequestTest extends TestCase
+{
+    /**
+     * @var PurchaseRequest
+     */
+    protected $request;
+
+    public function setUp()
+    {
+        $this->request = new PurchaseRequest($this->getHttpClient(), $this->getHttpRequest());
+        $this->request->initialize(
+            array(
+                'websiteKey' => 'web',
+                'secretKey' => 'secret',
+                'amount' => '12.00',
+                'returnUrl' => 'https://www.example.com/return',
+            )
+        );
+    }
+
+    public function testGetData()
+    {
+        $data = $this->request->getData();
+
+        $this->assertNotContains('Brq_payment_method', $data);
+        $this->assertNotContains('Brq_service_ideal_issuer', $data);
+        $this->assertNotContains('Brq_requestedservices', $data);
+    }
```

Comment:
@silvano-a 
Sure, good idea. Just improved the test. It will now check other data that is passed to the request as well.